nss-day-cohort-45/hello-world-column-counts
# Navbar Title Addition

## changes:
1) Added navbar title and wrapped it in a div with the navbar links
2) Imported mainBody.css into main.css
3) Added margin-bottom to header for a little additional spacing

## testing:
1) serve and load canada.html from this branch
2) make sure title appears on navbar
3) make sure main body styling appears correct (ensuring that import of mainBody.css into main.css was successful)
